src/main/scala/no/ndla/audioapi/AudioApiProperties.scala

```diff
@@ -40,7 +40,7 @@ object AudioApiProperties extends LazyLogging {
   val SearchServer = propOrElse("SEARCH_SERVER", "http://search-audio-api.ndla-local")
   val SearchRegion = propOrElse("SEARCH_REGION", "eu-central-1")
   val RunWithSignedSearchRequests = propOrElse("RUN_WITH_SIGNED_SEARCH_REQUESTS", "true").toBoolean
-  val SearchIndex = "audios"
+  lazy val SearchIndex = propOrElse("SEARCH_ALIAS", "audios")
```

Comment:
Må dette egentlig være en parameter? Antar det er for test purposes. I så fall kan vi ikke bare passe på å kalle indexen i test-elastic'en det samme som vi gjør "eksternt"?